templates/data/relational-databases.html

```diff
@@ -0,0 +1,294 @@
+			<p>
+				<a href="/contact-us" aria-controls="data-relational-dbs-modal" class="p-button--positive">Get in touch</a>	
+			</p>
+		</div>
+  </div>
+  <div class="p-suru" style="display: block;"></div>
+</section>
+
+<section class="p-section">
+	<div class="row--50-50">
+		<div class="col">
+			<h2>A system for managing digital information</h2>
+		</div>
+		<div class="col">
+			<p>Relational databases are systems to store, retrieve and manage digital information. They are one of many types of databases and, as their name implies, they enable us to express relations between groups of data.</p>
+			<p>Relational database management systems (RDBMS) organise data into tables. Each table is a set of rows where every cell has a type defined by its column. Each row is uniquely identified among all rows in the same table by a set of columns, called the primary key. A table can relate to another table by referencing its primary key. This relation is often called a foreign key. Foreign keys are used to express and enforce inclusion type relations between different tables at a database level.</p>
+			<p>For example, you can express that a table is the “child” of another one. The orders table can be considered a child of the customers table in the following diagram:</p>
+      <div class="p-strip is-shallow u-align--center u-vertically-center u-aspect-ratio--3-2" style="background-color: rgba(0,0,0,.03)">
+        {{
+					image (
+					url="https://assets.ubuntu.com/v1/d1ff2eae-1.jpg",
+					alt="",
+					width="1800",
+					height="1014",
+					hi_def=True,
+					loading="lazy"
+					) | safe
+				}}
+      </div>
+		</div>
+	</div>
+</section>
+
+<section class="p-section">
+  <hr class="p-rule is-fixed-width">
+	<div class="row--50-50">
+		<div class="col">
+			<h2>Relational model and relational databases</h2>
+		</div>
+		<div class="col">
+			<p>Relational databases are named after the relational model formalised by Edgar Frank Codd (also known as Ted Codd) in 1969. The diagram below maps the relational model terminology, as presented by Ted Codd, to RDBMS's terminology:</p>
+      <div class="p-strip is-shallow u-align--center u-vertically-center u-aspect-ratio--3-2">
+      {{
+				image (
+            url="https://assets.ubuntu.com/v1/4dc17519-2.jpg",
+            alt="",
+            width="1800",
+            height="1014",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+          }}
+      </div>
+			<p>The success of relational databases is rooted in the solid mathematical foundations of Ted’s model. Most relational database management systems use SQL to access the database. The versatility of the SQL language is another reason for the success of RDBMS.</p>
+		</div>
+	</div>	
+</section>
+
+<section class="p-section">
+	<hr class="is-fixed-width">
+	<div class="u-fixed-width p-section">
+		<h2>When to choose a relational database</h2>
+	</div>
+	<div class="row">
+		<div class="col-start-large-4 col-9">
```

Comment:
I assume we should add medium cols to the following 3 sections ("When to choose a relational database
", "The exciting journey of relational databases",  "Examples of popular relational databases
")

Reply:
I'd leave that to @lyubomir-popov to decide

Reply:
Done.